elements/pfe-navigation/src/pfe-navigation.scss

```diff
@@ -1086,6 +1086,30 @@ See mixin blocks before selectors that contain the selector name and the three l
     @include top-level-button--stays-in-top-bar;
   }
 }
+// @todo: (KS) add styles to hide account toggle and show the account link when logged-in class does NOT exist
+.pfe-navigation__account-wrapper {
+  
+  // Hide account toggle if NOT logged in
+  // .pfe-navigation__account-toggle {
+  //   display: none;
+  // }
+  // // Show login link if NOT logged in
+  // .pfe-navigation__log-in-link {
+  //   display: flex;
+  // }
+
+  .pfe-navigation__account-wrapper--logged-in {
```

Comment:
I've read some really interesting best practices lately around representing "state" such as "logged-in" via attributes instead of adding additional classes.  This makes it a lot easier to tell what is a state as well vs. an element.  If elements are always represented via class names and states represented as attributes, the entire stylesheet becomes a bit easier to parse.

Reply:
Sounds interesting, but this will have to be done separately from this PR since that's a large shift

Reply:
I'd also suggest that since the other components in the project use attributes to represent state, this is a big deviation to use classes for state instead.  This component needs to follow the coding conventions of the rest of the project if it wants to be merged into the main repo at some point.

Reply:
It should be done in a separate ticket is my point, thanks for calling it out.